contracts/optimistic-ethereum/iOVM/accounts/iOVM_ECDSAContractAccount.sol

```diff
@@ -14,11 +14,26 @@ interface iOVM_ECDSAContractAccount {
      * Public Functions *
      ********************/
 
+    /**
+     * Executes a signed transaction.
+     * @param _transaction Signed EOA transaction.
+     * @param _signatureType Hashing scheme used for the transaction (e.g., ETH signed message).
+     * @param _v Signature `v` parameter.
+     * @param _r Signature `r` parameter.
+     * @param _s Signature `s` parameter.
+     * @return Whether or not the call returned (rather than reverted).
+     * @return Data returned by the call.
+     */
```

Comment:
The same comment is in the implementation contract. Should we strive to have these comments in both places?